chandigarh-India.md

```diff
@@ -0,0 +1,5 @@
+# Chandigarh India
+- City Beautiful.. Cleanest city in India
+- First planned city in India
+- Awesome Punjabi food and boys
+
```

Comment:
Please elaborate on the food. Like best restaurants and their best dishes to taste